relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py

```diff
@@ -98,6 +100,7 @@ def run(
             "Command '%s' returned a non-zero return code, %d.%s%s"
             % (' '.join(e.cmd), e.returncode, stdout, stderr),
             1,
+            exit_on_error
```

Comment:
why is the previous call using an if and this one using an additional parameter ?